src/services/edusharing/index.js

```diff
@@ -25,20 +32,28 @@ class MerlinToken {
 
 module.exports = (app) => {
 	const eduSharingRoute = '/edu-sharing';
-	const merlinRoute = `${eduSharingRoute}/merlinToken`;
-	const docRoute = `${eduSharingRoute}/api`;
+	const eduSharingPlayerRoute = '/edu-sharing/player';
+	const merlinRoute = '/edu-sharing/merlinToken';
+	const docRoute = '/edu-sharing/api';
 
-	app.use(docRoute, staticContent(path.join(__dirname, '/docs/openapi.yaml')));
+	app.use(eduSharingRoute, new EduSharing(), (req, res) => {
+		res.send(res.data);
+	});
+	const eduSharingService = app.service(eduSharingRoute);
+	eduSharingService.hooks(hooks);
+
+	app.use(eduSharingPlayerRoute, new EduSharingPlayer(), (req, res) => {
```

Comment:
I understand why you add this new service and route to the legacy system. But i think add this to nestjs is only small more work, or migrate the complet service. We have strictly say nothing new should be add in legacy stack.